class-tgm-plugin-activation.php

```diff
@@ -2741,7 +2756,9 @@ protected function get_row_actions( $item ) {
 		 * @param object $item The current item.
 		 */
 		public function single_row( $item ) {
-			parent::single_row( $item );
+			echo '<tr class="' . esc_attr( strtolower( $item['type'] ) ) . '">';
```

Comment:
This `required` class is too generic - it may well clash with other plugins or maybe even core at some point in the future. Let's go for something like:

```
echo '<tr class="' . esc_attr( 'tgmpa-type-' . strtolower( $item['type'] ) ) . '">';
```

...and updating the CSS selector to match.
(With a prefixed class name, that may even allow the specificity of the selector to be dropped right down.)